test/cpp/jit/test_misc.cpp

```diff
@@ -2951,6 +2952,27 @@ graph(%x.1 : Tensor):
   }
 }
 
+TEST(TestFunctionExecutor, RunDecompositionTest) {
+  GraphFunction* func;
+  std::once_flag flag1;
+  for (bool unbiased : {true, false}) {
+    std::call_once(flag1, [&]() {
```

Comment:
just curious, why std::call_once and not just put this before the loop?

Reply:
This is mirroring how I expect the intended use case to go within functorch. You dont want to do the lookup from schema -> function and setting of execution on each call you want to run that code only once.